<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kvirtualview_base</class>
 <widget class="QWidget" name="kvirtualview_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KVirtual</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="gbCommon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Common</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_description">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2" colspan="3">
       <widget class="QTextEdit" name="textEdit_description">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_os">
        <property name="text">
         <string>Operating system</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBox_distrib">
        <item>
         <property name="text">
          <string notr="true">linux</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">archlinux</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>aptosid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">centos</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">debian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">fedora</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>fluxbuntu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">freebsd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">knoppix</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>kubuntu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">linuxmint</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">mandriva</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">mepis</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">openbsd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">redhat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">netbsd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">slackware</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">ubuntu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">windows</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QSvgWidget" name="widget_logo" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" rowspan="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="gbSystem">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>System</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_memory">
          <property name="text">
           <string>Base Memory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_memory">
          <property name="maximum">
           <number>1024</number>
          </property>
          <property name="singleStep">
           <number>128</number>
          </property>
          <property name="pageStep">
           <number>512</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntSpinBox" name="kintspinbox_memory">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
          <property name="singleStep">
           <number>128</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_nb_of_cpu">
          <property name="text">
           <string>Number of CPU</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_nb_of_cpu">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntSpinBox" name="kintspinbox_nb_of_cpu">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_usb">
        <property name="text">
         <string>USB Support</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_snapshot">
        <property name="text">
         <string>Snapshot</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="comboBox_boot">
        <item>
         <property name="text">
          <string>Disk</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CD-Rom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Floppy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Network</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_bootorder">
        <property name="text">
         <string>Boot on</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="gbDisplay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="4">
       <widget class="QComboBox" name="comboBox_video_card">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">std</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">cirrus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">vmware</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">qlx</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">xenfb</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_vga_type">
        <property name="text">
         <string>Video Card Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="4">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_none">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_direct">
          <property name="text">
           <string>Direct</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_vnc">
          <property name="text">
           <string>VNC</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_display_keyboard">
        <property name="text">
         <string>Keyboard type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="comboBox_display_keyboard">
        <property name="currentIndex">
         <number>14</number>
        </property>
        <property name="maxVisibleItems">
         <number>11</number>
        </property>
        <item>
         <property name="text">
          <string notr="true">ar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">de-ch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">es</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">fo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">fr-ca</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">hu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">ja</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">mk</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">no</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">pt-br</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">sv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">da</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">en-gb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">et</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">fr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">fr-ch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">is</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">lt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">nl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">pl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">ru</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">th</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">de</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">en-us</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">fi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">fr-be</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">hr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">it</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">lv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">nl-be</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">pt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">sl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">tr</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_vncport">
        <property name="text">
         <string>VNC Port</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="spinBox_vncport">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QGroupBox" name="gbNetwork">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Network</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget_ifaces">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Iface 1</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="KVirtualIfaceWidget" name="widget_iface0" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Iface 2</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="KVirtualIfaceWidget" name="widget_iface1" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Iface 3</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="KVirtualIfaceWidget" name="widget_iface2" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="gbStorage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Storage Media</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget_storage">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_1">
         <attribute name="title">
          <string>Storage Media 1</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="KVirtualStorageWidget" name="widget_storage0" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Storage Media 2</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="KVirtualStorageWidget" name="widget_storage1" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_8">
         <attribute name="title">
          <string>Storage Media 3</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <widget class="KVirtualStorageWidget" name="widget_storage2" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTextBrowser" name="textBrowser_output">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header>qsvgwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KVirtualIfaceWidget</class>
   <extends>QWidget</extends>
   <header>kvirtualifacewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KVirtualStorageWidget</class>
   <extends>QWidget</extends>
   <header>kvirtualstoragewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_vnc</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_vncport</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
